01-introduction.Rmd

```diff
@@ -19,24 +21,24 @@ In 2003, the {survey} package was released on CRAN and has been continuously dev
+There is one limitation to the {srvyr} package, the modeling functionality of the {survey} package is not ported over to tidy versions in the {srvyr} package. This book will use the {survey} package when discussing modeling and hypothesis testing, however, we will provide information on how to still pipe together the functions to make analyses that are easy to follow.
 
 What not to expect in this book:
 
 - Survey methodology - we only provide a primer on methodology
 - Statistical theory - we only provide a bit of formulas throughout
-- Weighting - we assume you are using an analysis-ready data file with weights
\ No newline at end of file
+- Weighting - we assume you are using an analysis-ready data file with weights
+
+What to expect in this book:
+
+- *Chapter \@ref(c02-overview-surveys)*: An overview of surveys and the process of designing surveys. This is only an overview, and we include many references to get more in-depth knowledge.
+- *Chapter \@ref(c03-specifying-sample-designs)*: Specifying sampling designs. Descriptions of common sampling designs, when they are used, the math behind the mean and standard error estimates, how to specify the designs in R, and examples using real data.
+- *Chapter \@ref(c04-understanding-survey-data-documentation)*: Understanding survey documentation. How to read the various components of survey documentation, working with missing data, and finding the documentation.
+- *Chapter \@ref(c05-descriptive-analysis)*: Descriptive analyses. Calculating point estimates along with their standard errors, confidence intervals, and design effects.
+- *Chapter \@ref(c06-statistical-testing)*: Statistical testing. Testing for differences between groups, including comparisons of means and proportions as well as goodness of fit tests, tests of independence, and tests of homogeneity.
+- *Chapter \@ref(c07-modeling)*: Modeling. Linear regression, ANOVA, and logistic regression.
+- *Chapter \@ref(c08-communicating-results): Communicating results. Developing analysis plans, describing results, reproducibility, making publishable tables and graphs, and helpful functions.
+- *Chapter \@ref(c09-ncvs-vignette)*: National Crime Victimization Survey Vignette. A vignette on how to analyze data from the NCVS, a survey in the US that collects information on crimes and their characteristics. This illustrates an analysis that requires multiple files to calculate victimization rates.
+- *Chapter \@ref(c10-ambarom-vignette)*: AmericasBarometer Vignette. A vignette on how to analyze data from the AmericasBarometer, a survey of attitudes, evaluations, experiences, and behavior in countries in the Western Hemisphere. This includes how to make choropleth maps with survey estimates.
```

Comment:
```suggestion
- *Chapter \@ref(c10-ambarom-vignette)*: AmericasBarometer Vignette. A vignette on how to analyze data from the AmericasBarometer, a survey of attitudes, evaluations, experiences, and behavior in countries in the Western Hemisphere. This includes how to make choropleth maps with survey estimates.

Most chapters include code to follow along with.  Each chapter with this code will start with a set-up section.  This will include all of the code needed to load packages and datasets to use in the chapter.  We then provide an overview of the topic along with examples on how to use the functions.  Most chapters end with exercises to work through.  Solutions to these exercises can be found online at [XX].
```